src/main/telemetry/frsky.h

```diff
@@ -22,10 +22,13 @@ typedef enum {
     FRSKY_VFAS_PRECISION_HIGH
 } frskyVFasPrecision_e;
 
+typedef void frSkyTelemetryInitFrameFunc(void);
```

Comment:
No problem, will fix. Will have to fix `pwm_output.h` as well, as this is what I used as a template.